### PR TITLE
Fixing Data Insertion Type Mismatch in Astro DB Table Definitions #7792

### PR DESCRIPTION
**Pull request by suzulang**
i fixed this issue
https://github.com/withastro/docs/issues/7792

### CODE REVIEW

**Comment by suzulang** [comment]
this is minimal-reproductions https://github.com/suzulang/minimal-reproduction

**Comment by liruifengv** [comment]
Thanks for you fix @suzulang! We only need to update the original content in English and leave it to our translators for other languages.

File: src/content/docs/en/guides/astro-db.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ import { defineTable, column } from 'astro:db';
 const Comment = defineTable({
   columns: {
     // A string of text.
```
Comment (liruifengv) [inline src/content/docs/en/guides/astro-db.mdx:89]: I think the comment need to be updated too

**Comment by liruifengv** [comment]
image
image

I found some similar questions with some code examples. Not sure if we want to update them. @sarah11918

**Comment by sarah11918** [comment]
Hi @suzulang !

Thank you for taking the time to contribute! As @liruifengv said, we only accept changes to the English documentation, but also, these examples are all individual and not meant to be used together. Each one describes individually what it does, and we did not write them to match with each other. Instead, some show different shapes/content so that people can see different examples of what a database might look like. These are not steps to follow, but individual sections to consult for reference.

So, I will close this PR as it is not an issue for us that these samples do not match. Instead, I think we should make sure we produce some content that does intend to show content linked together! I will ask our team to make a recipe to follow based on your issue linked here.

Thank you very much for bringing this up!

**Comment by suzulang** [comment]
> Hi @suzulang !
> 
> Thank you for taking the time to contribute! As @liruifengv said, we only accept changes to the English documentation, but also, these examples are all individual and not meant to be used together. Each one describes individually what it does, and we did not write them to match with each other. Instead, some show different shapes/content so that people can see different examples of what a database might look like. These are not steps to follow, but individual sections to consult for reference.
> 
> So, I will close this PR as it is not an issue for us that these samples do not match. Instead, I think we should make sure we produce some content that does intend to show content linked together! I will ask our team to make a recipe to follow based on your issue linked here.
> 
> Thank you very much for bringing this up!

Hello    @liruifengv  and team,

Thank you very much for your prompt response and clarification. I understand the guidelines presented and the explanation regarding the individual examples. I will keep an eye on the updates and new content, especially looking forward to those recipes that intend to show linked content. I am glad to contribute to the improvement of the documentation, and I will be keen to participate more actively if there are more suitable opportunities for contributions in the future.

Thank you again for your hard work and feedback, looking forward to seeing more excellent content.

Best regards

**Pull request closed without merging**